alignak/http/generic_interface.py

```diff
@@ -108,16 +108,22 @@ def have_conf(self, magic_hash=None):  # pylint: disable=W0613
         return self.app.cur_conf is not None
 
     @cherrypy.expose
+    @cherrypy.tools.json_in()
     @cherrypy.tools.json_out()
-    def set_log_level(self, loglevel):  # pylint: disable=R0201
+    def set_log_level(self, loglevel=None):  # pylint: disable=R0201
         """Set the current log level in [NOTSET, DEBUG, INFO, WARNING, ERROR, CRITICAL, UNKNOWN]
 
         :param loglevel: a value in one of the above
         :type loglevel: str
         :return: None
         """
+        if loglevel is None:
+            parameters = cherrypy.request.json
+            loglevel = parameters['loglevel']
```

Comment:
Or it means that the json parameters are FORM parameters and not JSON BODY payload.